commands/anonSuggest.js

```diff
@@ -0,0 +1,39 @@
+const {GuildChannel} = require('discord.js');
+
+module.exports = {
+	name: 'anonsuggest',
+	description: 'Posts an anonymous Suggestion',
+	usage: 'anonsuggest',
+	helpMsg: true,
+	execute(message, args) {
+		const {modBot} = message.client.config.channelIDs;
+		const suggestion = args.join(' ');
+
+		const lastChar = args[0].slice(args[0].length - 1, args[0].length);
+		let number;
+
+		if (!isNaN(lastChar)) {
+			number = args[0];
+		}else {
```

Comment:
spacing bruh, adding the rule into linter now

Reply:
Fixed